torchx/Makefile

```diff
@@ -5,11 +5,29 @@ TORCHX_SO = $(PRIV_DIR)/torchx.so
 # CMAKE_BUILD_FLAGS = --verbose
 CMAKE_BUILD_DIR ?= $(MIX_APP_PATH)/cmake
 C_SRC = $(shell pwd)/c_src
+NERVES_BUILD ?= nerves_build_no
+
+# this .cmake file makes this project compatible with nerves and it should have no effect on normal build
+TOOLCHAIN_FILE ?= $(shell pwd)/nerves/toolchain.cmake
```

Comment:
This seems to be nerves-specific as well, so perhaps this default can be removed?

Reply:
From what I can tell, if this isn't present, everything should work normally, right? So we can just remove this default.

It seems that a good compromise would be adding a section on the Torchx README regarding Nerves build